ci/tests.yaml

```diff
@@ -19,3 +19,4 @@ dependencies:
   - setuptools_scm
   - pip:
     - pyradiosky
+    - git+https://github.com/aelanman/lunarsky.git
```

Comment:
It would be good to get lunarsky on pypi sometime soon (not required for this PR).

Reply:
I added lunarsky to PyPi. No reason not to at this point.

Reply:
great!